src/components/PeopleFilters.tsx

```diff
@@ -1,27 +1,92 @@
+import React, { useContext } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { getSearchWith } from '../utils/searchHelper';
+import { DispatchContext } from '../context/reducer';
+import { SearchLink } from './SearchLink';
+
+enum SexFilter {
+  'male',
+  'female',
+}
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const centuries = searchParams.getAll('centuries') || [];
+  const dispatch = useContext(DispatchContext);
+
+  const handleClickFilter = (sortParam: SexFilter | null) => {
+    switch (sortParam) {
+      case SexFilter.male:
+        return setSearchParams(getSearchWith(searchParams, { sex: 'm' }));
+      case SexFilter.female:
+        return setSearchParams(getSearchWith(searchParams, { sex: 'f' }));
+      default:
+        return setSearchParams(getSearchWith(searchParams, { sex: null }));
+    }
+  };
```

Comment:
and then you can simplify this function
```suggestion
  const handleClickFilter = (sortParam: SexFilter | null) => {
    setSearchParams(getSearchWith(searchParams, { sex: sortParam }));
  };
```